addon/modifiers/style.js

```diff
@@ -59,8 +59,10 @@ function setStyles(element, newStyles, oldStyles) {
   }
 }
 
-export default Ember._setModifierManager(
+export default setModifierManager(
   () => ({
+    capabilities: capabilities('3.13'),
```

Comment:
How does this work together with the polyfill? If the `ember-modifier-manager-polyfill` supports that API at all. As already that above, dropping support for Ember < 3.13 would be also fine for me.

Reply:
We don't need to drop Ember < 3.13 as `ember-modifier-manager-polyfill` polyfills capabilities API as well.